client/src/shared/components/Text-field/text-field.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+interface Props {
+  label?: string;
+  placeholder?: string;
+  width: number;
+  height: number;
+  className: string;
+  resizable: boolean;
+}
+
+const Textfield: React.FC<Props> = ({ label, placeholder, width, height, className, resizable }: Props) => {
+  return (
+          <div className="p-5 h-auto w-auto">
+            {label !== '' ? (<div><label>{label} </label></div>) : null}
```

Comment:
- it will be more readable pag naka multiline.
- as much as possible limit empty conditions by using conditional operators

```suggestion
            {label !== null && (
                 <div>
                     <label>{label}</label>
                 </div>
             )}
```